packages/client/src/arpa_reporter/views/NewUpload.vue

```diff
@@ -1,111 +1,164 @@
 <template>
-  <div class="upload">
-    <h1>Submit Workbook</h1>
+  <div>
 
-    <div>
-      <div v-if="error" class="mt-3 alert alert-danger" role="alert">
-        {{ error }}
-      </div>
-
-      <form
-        method="post"
-        encType="multipart/form-data"
-        ref="form"
-        @submit.prevent="onSubmit"
-      >
-        <div class="form-group">
-          <input
-            class="form-control-file"
-            type="file"
-            id="spreadsheet"
-            name="spreadsheet"
-            @change="changeFiles"
-            ref="files"
-          />
-        </div>
-        <div class="form-group">
-          <button
-            class="btn btn-primary"
-            type="submit"
-            :disabled="uploadDisabled"
-            @click.prevent="onSubmit"
-          >
-            {{ uploadButtonLabel }}
-          </button>
-          <a class="ml-3" href="#" @click="cancelUpload">Cancel</a>
-        </div>
-      </form>
-    </div>
+    <FormulateForm
+      name="new-upload"
+      :form-errors="formErrors"
+      v-model="values"
+      :schema="schema"
+      @submit="onSubmit"
+    >
+    <FormulateErrors
+      class="alert alert-danger"
+    />
+    </FormulateForm>
   </div>
 </template>
 
 <script>
+import Vue from 'vue';
+import VueFormulate from '@braid/vue-formulate';
 import { postForm } from '../store';
 
+Vue.use(VueFormulate, {
+  classes: {
+    prefix: 'formulate-',
```

Comment:
What is this prefix class for?  It doesn't obviously change anything in my testing.

Reply:
Ah, interesting - this is actually doubly wrong. 

I believe I had seen a reference to this in an issue in the repo, as a means of prefixing class names so as to avoid conflicts with other library class names. The property is apparently actually named `idPrefix`, and is a sibling to `classes`. 

Also though, by [default](https://github.com/wearebraid/vue-formulate/blob/3fdd6fdc2ed82113e6a78dfeec800b36763d7f95/src/Formulate.js#L96) it's `formulate-`, so i can just remove this line.